docs/source/getting-started.rst

```diff
@@ -87,6 +87,10 @@ with permissions for at least the following services:
 
 Deploy
 ------
+.. note:: Streamalert supports Terraform's native ability to lock the remote s3 state file whenever a user is planning and applying Terraform configuration.
```

Comment:
`Streamalert` --> `StreamAlert`

Reply:
🤦‍♂ I double checked and still missed one